README.md

```diff
@@ -1100,6 +1100,15 @@ spctl --add /path/to/Application.app
 spctl --remove /path/to/Application.app
 ```
 
+#### Allow Applications from Anywhere
+```bash
+# Disable Gatekeeper
```

Comment:
Lose the "Gatekeeper" and just leave "Enable" and "Disable". Also add a `(Default)` statement behind the default setting.